forge/test/mlir/test_ops.py

```diff
@@ -89,6 +89,40 @@ def forward(self, a):
     co_out = [co.to("cpu") for co in co_out]
     assert compare_with_golden_pcc(golden=fw_out, calculated=co_out[0], pcc=0.99)
 
+@pytest.mark.parametrize("inputs_and_dim",
+    [((2, 2, 32, 32), (2, 2, 32, 32), 0),
```

Comment:
Can we also check some non-tile dim-aligned shapes for concat? Is that supported on ttnn? 

If not, just leave a comment in the test.